Source/WTF/wtf/MemoryPressureHandler.cpp

```diff
@@ -106,7 +106,7 @@ static ASCIILiteral toString(MemoryUsagePolicy policy)
 }
 #endif
 
-static size_t thresholdForMemoryKillOfActiveProcess(unsigned tabCount)
+static size_t thresholdForMemoryKillOfActiveProcessWithPageCount(unsigned tabCount)
```

Comment:
Should we rename the parameter to `pageCount` to better match the function name?

Reply:
I am also wondering if it may make sense to rename the function to e.g. `memoryThresholdForTerminationOfActiveProcessWithPageCount` or just `memoryThresholdOfActiveProcessWithPageCount`?